wallet-new/integration/Functions.hs

```diff
@@ -162,6 +176,69 @@ runAction wc ws CreateAddress = do
         , newaddrWalletId         = wId
         }
 
+-- Transactions
+runAction wc ws CreateTransaction = do
+
+    let localAccounts  = ws ^. accounts
+    let localAddresses = ws ^. addresses
+
+    let localAccsWithMoney = filter ((> minBound) . accAmount) localAccounts
+
+    -- | The preconditions we need to generate a transaction.
+    -- We need to have an account and two addresses.
+    -- We also need money to execute a transaction.
+    guard (length localAccounts       >= 1)
+    guard (length localAddresses      >= 2)
+    guard (length localAccsWithMoney  >= 1)
+
+    -- From which source to pay.
+    accountSource <- pickRandomElement localAccsWithMoney
+
+    let _accountSourceMoney = accAmount accountSource
+
+    -- We should probably have a sensible minimum value.
+    -- moneyAmount <- liftIO $ mkCoin $ generate $ choose (0, getCoin accountSourceMoney)
```

Comment:
yes, with
```haskell
 moneyAmount <- liftIO $ generate arbitrary
```
creating transaction will probably fail due too fees. I believe your intuition is right here to  generate something minimum for fee to be included.